### DISCUSSION TITLE
I have a question

### DISCUSSION THREAD

**Discussion by HwiPle** [web3j/web3j#1553] [Q&A]
java.lang.NoClassDefFoundError: Failed resolution of: Lorg/web3j/protocol/http/HttpService;

AndroidJavaException: java.lang.NoClassDefFoundError: Failed resolution of: Lorg/web3j/protocol/http/HttpService;

java.lang.ClassNotFoundException: Didn't find class "org.web3j.protocol.http.HttpService" on path: DexPathList[[zip file "/data/app/~~G9VPkrcMFlQ-biOyrvUp-Q==/com.DefaultCompany.AndroidTestMode-OZCzOyc11V090KjT1cNipQ==/base.apk"],nativeLibraryDirectories=[/data/app/~~G9VPkrcMFlQ-biOyrvUp-Q=

Problematic code : Web3j.build(new HttpService(url));

I just want to be connected

I am making it in android studio.

Does anyone know how to solve it?

**Comment by jambestwick** [comment]
which andoid version  of your phone device？ nmaybe try a before android 9.0 ，org/web3j/protocol/http/HttpService it's not very fit up 9.0
